R/mzQCjson.R

```diff
@@ -0,0 +1,255 @@
+library(stringr)
```

Comment:
no need to use library here (if you prefix all functions which are used from these two libraries with their namespace, e.g. `stringr::....`